docs/quick_start/viewing.rst

```diff
@@ -46,6 +49,26 @@ It is also possible to change the default duration of a graph when it is first d
 
 This can be useful when very high frequency data is ingested into the system as it will prevent the initial graph that is displayed from pulling large amounts of data from the system and slowing down the response of the system and the GUI.
 
+When an asset does not continuously ingest data you may need to move back in time in order to see the last data that was ingested for an ingest. The interface provides a convenient shortcut to allow you to quickly navigate back in time to see the last data that was ingested. Simply click in the icon to the left of the time navigation buttons.
+
++---------------+
+| |latest_icon| |
++---------------+
+
+The graph will change to show the latest data available for the chosen asset.
+
++----------------+
+| |latest_graph| |
++----------------+
+
+Notice that the icon used to get this graph has now changed.
+
++----------------+
+| |current_icon| |
++----------------+
+
+Clicking on this icon will cause the graph to run to the current time frame and show any data that is freshly ingested.
+
```

Comment:
Can we explain pause/play, previous, next buttons i.e about `time navigation buttons`?